Formula/fanyi.rb

```diff
@@ -0,0 +1,20 @@
+require "language/node"
+
+class Fanyi < Formula
+  desc "🇨🇳 and 🇺🇸 translate tool in your command-line"
```

Comment:
Please just use words here
```suggestion
  desc "Mandarin and english translate tool in your command-line"
```